api/controllers/AuthController.js

```diff
@@ -68,12 +68,8 @@ module.exports = {
    * @param {Object} res
    */
   logout: function (req, res) {
-    req.logout();
-
-    // mark the user as logged out for auth purposes
-    req.session.authenticated = false;
-
-    res.redirect('/');
+    req.session.destroy();
```

Comment:
Passport.js seems to behave a bit strangely with requests that come from websockets -- the req.logout function doesn't end up attached.

Also, using `res.redirect` doesn't work in general with websockets, so it returns a 200 instead and this tells the client to do a redirect.